lisp/mastodon-tl.el

```diff
@@ -415,10 +416,10 @@ links in the text."
            maybe-userhandle
            ;; Until we have native profile page viewing we leave these as HTML
            ;; links.
-           (setq mastodon-tab-stop-type 'shr-url
-                 keymap mastodon-tl--shr-map-replacement
-                 help-echo (concat "Browse user profile of " maybe-userhandle)))
-
+           (setq mastodon-tab-stop-type 'handle
+                 keymap mastodon-tl--link-keymap
+                 help-echo (concat "Browse user profile of " maybe-userhandle)
+                 extra-properties (list 'mastodon-handle maybe-userhandle)))
```

Comment:
We can also look up the id in the toot's `mention` field.
With that we still have to go to the server to look up the full account details but getting details for a specific account (via [`GET /api/v1/accounts/:id`](https://github.com/tootsuite/documentation/blob/master/Using-the-API/API.md#accounts)) should be much lighter on the server.

Reply:
That would work for byline handles, but do inline handles have id's in their meta info? For the time being we will look up the account id in 'toot-json, if it doesn't exist we can do a search.

Reply:
I see, the ids are in the mentions, will save a lot of search calls.